SDL_ttf.h

```diff
@@ -426,6 +413,32 @@ extern DECLSPEC SDL_bool TTF_GetFontSDF(const TTF_Font *font);
 #define TTF_SetError    SDL_SetError
 #define TTF_GetError    SDL_GetError
 
+/**
+ * \brief Direction
+ */
+typedef enum
+{
+  TTF_DIRECTION_LTR = 0,
+  TTF_DIRECTION_RTL,
+  TTF_DIRECTION_TTB,
+  TTF_DIRECTION_BTT
+} TTF_Direction;
+
+/* Set Direction and Script to be used for text shaping.
+   This functions returns always 0, or -1 if SDL_ttf is not compiled with HarfBuzz
+
+   Deprecated. Prefer TTF_SetFontDirection() and TTF_SetFontScriptName()
+*/
+extern DECLSPEC int SDLCALL TTF_SetDirection(int direction); /* hb_direction_t */
+extern DECLSPEC int SDLCALL TTF_SetScript(int script); /* hb_script_t */
+
+/* Set direction and script per font 
+   'script' is null terminated string of exactly 4 characters.
+   This functions returns 0, or -1 if SDL_ttf is not compiled with HarfBuzz or invalid parameter
```

Comment:
`These functions return 0, or -1 if SDL_ttf is not compiled with HarfBuzz or in case of an invalid parameter.`